core/extensions.rs

```diff
@@ -17,6 +17,7 @@ pub struct OpDecl {
   pub enabled: bool,
   pub is_async: bool, // TODO(@AaronO): enum sync/async/fast ?
   pub is_unstable: bool,
+  pub is_v8: bool,
```

Comment:
`is_async`/`is_v8` could actually be an enum - these are mutually exclusive

Reply:
Yeah, it's hinted at in the above TODO, will look into enforcing it once I land fast ops which I've mostly prototyped locally.